DQMServices/StreamerIO/plugins/BuildFile.xml

```diff
@@ -8,4 +8,5 @@
 
 <library   file="*.cc" name="DQMServicesStreamerIOPlugins">
   <flags   EDM_PLUGIN="1"/>
+  <flags   REM_CXXFLAGS="-Werror=unused-variable"/>
```

Comment:
Hi David,

because otherwise the build fails in a boost include:
/opt/offline/slc6_amd64_gcc491/external/boost/1.51.0-cms2/include/boost/asio/error.hpp:244:45: error: 'boost::asio::error::system_category' defined but not used [-Werror=unused-variable]
 static const boost::system::error_category& system_category

Dmitrijus